src/rules/incognia-authentication.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * @title Incognia Authentication Rule
+ * @overview Verify if the device logging in is at a trusted location.
+ * @gallery true
+ * @category marketplace
+ *
+ */
+async function incogniaAuthenticationRule(user, context, callback) {
+  const _ = require('lodash@4.17.19')
+
+  const { IncogniaAPI } = require('@incognia/api@1.0.0')
+  const { Auth0UserUpdateUtilities } = require('@auth0/rule-utilities@0.2.0')
+
+  const { INCOGNIA_CLIENT_ID, INCOGNIA_CLIENT_SECRET } = configuration;
```

Comment:
Just to make sure ... are these the same credentials as the other Rule? In other words, if I have both installed it would be expected that the credentials are the same?